lib/checks/mobile/target-size.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "target-size",
+  "evaluate": "target-size-evaluate",
+  "options": {
+    "minSpacing": 24
```

Comment:
`minSpacing` doesn't make much sense for target size. This should be `minSize` or `targetSize`

Reply:
Agreed, should be `minSize`.